config/routes.rb

```diff
@@ -1,7 +1,7 @@
 Rails.application.routes.draw do
   root to: 'posts#index'
-  resources :comments
-  resources :posts
+  resources :posts do
+    resources :comments
```

Comment:
if u remove index and new action from controller u should delete them from routes